config/routes.rb

```diff
@@ -65,5 +67,5 @@
     get "*any", to: "error#handler"
   end
 
-  root to: 'homepage#index', via: :get
+  match "*any", to: proc { |env| [405, {}, ["Not Allowed"]] }, via: :all
```

Comment:
Hmm, on 2nd thought, this will also return 405 for GET requests for pages that don't exist, so 405 isn't the correct status code.

> The HyperText Transfer Protocol (HTTP) 405 Method Not Allowed response status code indicates that the request method is known by the server but is not supported by the target resource.

https://httpstatusdogs.com/405-method-not-allowed

Reply:
But those shouldn't get to frontend, so is that a problem?

Reply:
I thought about listing the verbs individually, so happy to do that.  Some posts are legit too though, so 🤷‍♂️